styleguide/public/kss.js

```diff
@@ -0,0 +1,53 @@
+(function() {
+  var KssStateGenerator;
+
+  KssStateGenerator = (function() {
+    var pseudo_selectors;
+
+    pseudo_selectors = ['hover', 'enabled', 'disabled', 'active', 'visited', 'focus', 'target', 'checked', 'empty', 'first-of-type', 'last-of-type', 'first-child', 'last-child'];
+
+    function KssStateGenerator() {
+      var idx, idxs, pseudos, replaceRule, rule, stylesheet, _i, _len, _len2, _ref, _ref2;
+      pseudos = new RegExp("(\\:" + (pseudo_selectors.join('|\\:')) + ")", "g");
```

Comment:
Mixed double and single quotes.<br>Identifier 'pseudo_selectors' is not in camel case.